client/entrypoints/express-checkout/tracking.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { debounce } from 'lodash';
+import { recordUserEvent } from 'tracks';
```

Comment:
Should be using `import { recordEvent } from 'wcstripe/tracking';` here.

Reply:
Fixed in [8956e8e](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3436/commits/8956e8e9ebd2217c7998e16d3cc6c3286bf40296)